third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -57,17 +57,18 @@ def arcs_kt_binary(name, srcs = [], deps = [], visibility = None):
 
         deps = [":" + libname]
 
+    bin_name = name + "_bin"
     kt_native_binary(
-        name = name,
+        name = bin_name,
         entry_point = "arcs.main",
         deps = [_to_wasm_dep(dep) for dep in _ARCS_KOTLIN_LIBS + deps],
-        tags = ["wasm"],
+        tags = ["manual"],
```

Comment:
This tag won't really have any effect, since `name` depends on it. `bazel build ...` will cause `name` to be built, and `name` depends on `bin_name`, so `bin_name` will also be built.

I think you can just delete this line.

Reply:
I tested this line and it made the difference between working and not working.

Reply:
https://g3doc.corp.google.com/testing/integration/guitar/g3doc/guitar-3/encyclopedia/developer/build.md?cl=head#common-tags

Reply:
In what way did it not work?

Does the `name` target get built? How does that work if it doesn't build its dependency?